### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by eofafrica4lyf**
Here's the link to the screenshot of the coverage sir.
http://prntscr.com/nobf88

### CODE REVIEW

**Review by Austine3000 — COMMENTED** [review]
Please add a screenshot of your test coverage.

Here's the screenshot sir
image

File: alternatingCase/alternatingCase.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+require("../toUpper/toUpper");
```
Comment (Austine3000) [inline alternatingCase/alternatingCase.js]: Change this to a single quote. Look at Airbnb Javascript style guide. Make this correction across your code.

File: alternatingCase/alternatingCase.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+require("../toLower/toLower");
+
+//AlternatingCase returns the letters in alternating cases starting with a lowercase
+String.prototype.alternatingCase = function() {
+  var alternatingCase = "";//final output initialized to an empty string
```
Comment (Austine3000) [inline alternatingCase/alternatingCase.js]: This style of commenting is against AirBnB style. Comments should be above the code it's intended for. Make this correction across the code

File: alternatingCase/alternatingCase.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  var regexTest = /[a-z]/i;//regex pattern for letters whether in uppercase or lowercase
+  for (var index = 0; index < this.length; index++) {
+    //On each character of string, we check that it is a string character and perform the toLower and toUpper operations alternately
```
Comment (Austine3000) [inline alternatingCase/alternatingCase.js]: There should be space after the comment symbol, do this across your code.

File: alternatingCase/alternatingCase.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+require("../toUpper/toUpper");
+require("../toLower/toLower");
+
+//AlternatingCase returns the letters in alternating cases starting with a lowercase
+String.prototype.alternatingCase = function() {
```
Comment (Austine3000) [inline alternatingCase/alternatingCase.js]: Give a space between function and ().

File: alternatingCase/alternatingCase.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+  var regexTest = /[a-z]/i;//regex pattern for letters whether in uppercase or lowercase
+  for (var index = 0; index < this.length; index++) {
+    //On each character of string, we check that it is a string character and perform the toLower and toUpper operations alternately
+    alternatingCase +=
+      regexTest.test(this[index]) === true && index % 2 === 0
+        ? this[index].toLowerCase()
+        : this[index].toUpperCase();
+  }
+  return alternatingCase;
+};
```
Comment (Austine3000) [inline alternatingCase/alternatingCase.js:18]: At the end of every file add a one line space

**Review by Austine3000 — COMMENTED** [review]
Please read the AirBnB Javascript guide and remember to add a screenshot of your code coverage.

**Comment by eofafrica4lyf** [comment]
All done sir.

**Pull request merged**